examples/pio_blink/rpx0xx_pio_blink/blink.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2023 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <errno.h>
+#include "pio/pio.h"
+#include "blink.hex.pio.h"
+
+static void pio_blink_init_pins(pio_t pio, pio_sm_t sm, gpio_t pin)
+{
+    pio_gpio_init_t init = {
+        .gpio_direction = 0x01,
```

Comment:
0x01 is output?

Reply:
`base` and `count` are probably clear and obvious. `state` and `direcetion` are bitmasks corresponding to the mapped pins. Least significant bit in the mask shall map to the first mapped pin. 0/1 in state mean off/on, 0/1 in direction meas input/output.

Reply:
You could try it out in your program  to test them if you want.

Reply:
A define for the bit masks would make this more readable